### PR TITLE
tests: Fix Unexpected error in InvalidMemoryAliasing

### PR DESCRIPTION
**Pull request by renelindsay**
Fixed VkLayerTest.InvalidMemoryAliasing

Fixes issue #1280 .

Change-Id: Ic6f7ade06c35eb45f671e19c5cfd2ada0b2daa4f

### CODE REVIEW

File: tests/layer_validation_tests.cpp [hunk 1]
```diff
@@ -7450,6 +7454,7 @@ TEST_F(VkLayerTest, PSOViewportCountWithoutDataAndDynScissorMismatch) {
     VkPipelineRasterizationStateCreateInfo rs_ci = {};
     rs_ci.sType = VK_STRUCTURE_TYPE_PIPELINE_RASTERIZATION_STATE_CREATE_INFO;
     rs_ci.pNext = nullptr;
+    rs_ci.lineWidth = m_device->props.limits.lineWidthRange[0];
```
Comment (tobine) [inline tests/layer_validation_tests.cpp]: Was this included by accident? Should be in separate CL or update description of this CL to encompass this test as well if change was intended.

**Comment by renelindsay** [comment]
@tobine I have made the changes you requested.  Is this ok?
